revolution_pi/gpio_chip.go

```diff
@@ -17,16 +17,17 @@ type gpioChip struct {
 	dev        string
 	logger     logging.Logger
 	fileHandle *os.File
+	dioDevices []SDeviceInfo
```

Comment:
Is each DIO device a physical add on to the board?

Reply:
yep, a user can add multiple DIO modules if they wanted